CorsixTH/corsix-th.6

```diff
@@ -0,0 +1,84 @@
+.\" Copyright (c) 2018 Stephen "TheCycoONE" Baker
+.\"
+.\" Permission is hereby granted, free of charge, to any person obtaining a copy of
+.\" this software and associated documentation files (the "Software"), to deal in
+.\" the Software without restriction, including without limitation the rights to
+.\" use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies
+.\" of the Software, and to permit persons to whom the Software is furnished to do
+.\" so, subject to the following conditions:
+.\" 
+.\" The above copyright notice and this permission notice shall be included in all
+.\" copies or substantial portions of the Software.
+.\" 
+.\" THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+.\" IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+.\" FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+.\" AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+.\" LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+.\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+.\" SOFTWARE.
+.\"
+.\" This manpage is written in mdoc(7).
+.\" Language reference: https://man.openbsd.org/mdoc.7
+.Dd August 22, 2018
+.Dt CORSIX-TH 6
+.Os
+.Sh NAME
+.Nm corsix-th
+.Nd "An" open source implementation "of" Theme Hospital
+.Sh SYNOPSIS
+.Nm
+.Op Fl -bitmap-dir Ns Ar =path
```

Comment:
Is it intentional that the `=` is concatenated with `path` here? other options do it differently, not to mention line 44